### PR TITLE
Revert "Move steam.js to unpublishedScripts"

### PR DESCRIPTION
**Pull request by Atlante45**
We'd disabled some Steam related UI until we released, time to turn it back on.

You'll need Steam to test this.

Test plan:
- Install the Steam `test-build` beta.
- Ping me or get a friend to do the same
- Invite each other in different scenarios using the Steam invite button in the hud bar.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 799f1b8737e1b61ea6ea334e887c88933d613542 available at: https://gist.github.com/6ea388768fd99d2ab4a4616e0b8f5b3a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9005-799f1b8.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9005-799f1b8.dmg

File: scripts/defaultScripts.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ var DEFAULT_SCRIPTS = [
     "system/mute.js",
     "system/goto.js",
     "system/hmd.js",
+   "system/steam.js",
```
Comment (huffman) [inline scripts/defaultScripts.js]: This needs 1 more space of indentation :P

Comment (Atlante45) [inline scripts/defaultScripts.js]: Dang it, fixed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 265effd1f1c581610c87f45d28e259c58530b2a0 available at: https://gist.github.com/6ea388768fd99d2ab4a4616e0b8f5b3a<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9005-265effd.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9005-265effd.dmg

**Comment by kevinmthomasse** [comment]
## QA Test
**QA Tester**: KevinMThomas
**QA Pass**: **QA Pass** FAILURE
**Additional Test Results + Observations**: Failure. Tested with @Clement as the invite feature was not working properly. When I clicked on the UI nothing occurred.

**Pull request closed without merging**